.prettierrc.json

```diff
@@ -1,9 +1,7 @@
 {
-    "plugins": ["prettier-plugin-organize-imports"],
     "singleQuote": true,
     "trailingComma": "all",
     "tabWidth": 4,
-    "organizeImportsSkipDestructiveCodeActions": true,
```

Comment:
had to disable `prettier-plugin-organize-imports` because after upgrading the TS somehow formatting reported more than ~300 files with unsorted imports 🙃

Reply:
if so, can you remove the package from package json?